nginx/amiable.sh

```diff
@@ -5,15 +5,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 nginxHome=`nginx -V 2>&1 | grep "configure arguments:" | sed 's/[^*]*conf-path=\([^ ]*\)\/nginx\.conf.*/\1/g'`
 
-SYSTEM=$(uname -s)
-if [ $SYSTEM == "Darwin" ]; then
-    # Mac OS X platform
-    confDir=servers
-elif [ $SYSTEM == "Linux" ]; then
-    confDir=sites-enabled
-fi
-
-sudo ln -fs $DIR/amiable.conf $nginxHome/$confDir/amiable.conf
+sudo ln -fs $DIR/amiable.conf $nginxHome/sites-enabled/amiable.conf
```

Comment:
Why removing that? On my instance with `homebrew`, `servers` directory is included in the default configuration instead of `sites-enabled`, i.e
```
include servers/*;
```

Reply:
hmm ok. On my mac its called sites-enabled which meant this didn't work. I'll put it back or try and make it smarter!